Cubical/HITs/Truncation/FromNegOne/Base.agda

```diff
@@ -13,8 +13,17 @@ open import Cubical.Data.NatMinusOne renaming (suc₋₁ to suc)
 open import Cubical.Foundations.Prelude
 open import Cubical.Foundations.HLevels
 open import Cubical.HITs.Sn
+open import Cubical.Data.Nat.Base renaming (suc to sucℕ)
+open import Cubical.Data.Unit.Base
 
-data  ∥_∥_ {ℓ} (A : Type ℓ) (n : ℕ₋₁) : Type ℓ where
-  ∣_∣ : A  → ∥ A ∥ n
-  hub : (f : S (suc n) → ∥ A ∥ n) → ∥ A ∥ n
-  spoke : (f : S (suc n) → ∥ A ∥ n) (x : S (suc n)) → hub f ≡ f x
+data  ∥_∥⁻¹_ {ℓ} (A : Type ℓ) (n : ℕ₋₁) : Type ℓ where
```

Comment:
I would not introduce the ^-1 here. Maybe better to just call it `HubAndSpoke` or something?